File: problems/l/ladder/ladder.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

from math import ceil, sin, radians


def main():
    h, v = [int(val) for val in input().split()]

    print(ceil(h / sin(radians(v))))

if __name__ == '__main__':
    main()

File: problems/l/ladder/ladder.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>
#include <cmath>

using namespace std;


double to_radians(double degrees) {
    return (degrees * M_PI ) / 180.0;
}

int main() {
    int h, v, ladder;

    cin >> h >> v;

    ladder = ceil(h / sin(to_radians(v)));

    cout << ladder << endl;
}